protocol/kafka_confluent/v2/message.go

```diff
@@ -0,0 +1,143 @@
+/*
+ Copyright 2023 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package kafka_confluent
+
+import (
+	"bytes"
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/cloudevents/sdk-go/v2/binding"
+	"github.com/cloudevents/sdk-go/v2/binding/format"
+	"github.com/cloudevents/sdk-go/v2/binding/spec"
+	"github.com/confluentinc/confluent-kafka-go/v2/kafka"
+)
+
+const (
+	prefix         = "ce-"
+	contentTypeKey = "Content-Type"
```

Comment:
question: why not make it lower here to directly?

Reply:
No particular reason to use the capital. I thought the `contety-type` might be a special property to decide the event format.

To align it with the sarama protocol. I changed it to lowercase.